routes/classRouter.js

```diff
@@ -0,0 +1,104 @@
+const classRouter = require("express").Router();
+const mySchool = require("./../School.js");
+const displayTime = require("./../build.js");
+
+classRouter.get("/", (req, res) => {
+  res.json(mySchool.classes);
+});
+
+classRouter.post("/", (req, res) => {
+  res.json(req.body);
+});
+
+classRouter.post("/add", (req, res) => {
+  let name = req.body.name;
+  let teacher = req.body.teacher;
+
+  if (mySchool.classes[name]) {
+    res.json({
+      error: "Please fill out all the information or Class already exists",
+      timestamp: displayTime()
+    });
+  } else {
+    mySchool.addClass(name, teacher);
+
+    res.json({
+      class: mySchool.classes[name],
+      message: "Created a new class",
+      status: "ok",
+      timestamp: displayTime()
+    });
+  }
+});
+
+classRouter.post("/:className/enroll", (req, res) => {
+  let className = req.params.className;
+  let studentName = req.body.name;
+  let studentAge = req.body.age;
+  let studentCity = req.body.city;
+  let studentGrade = req.body.grade;
+  let studentsArr = mySchool.classes[className].students;
+
+  studentsArr.forEach(student => {
```

Comment:
Okay so, since to start off with no students, the forEach loop wont run at all, so anything inside the forEach loop dont ever happen.